### DISCUSSION TITLE
[Markdown] Dark mode of image in github wiki page markdown is not working

### DISCUSSION THREAD

**Discussion by ghost-code-ninja** [community/community#57250] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

I'd like to use the different image for the dark mode in github wiki markdown page but it's not working.
Looks like it's working only on github readMe.md file. Hope someone help me with this.

```
![Fancy logo](./dark.png#gh-dark-mode-only)
![Fancy logo](./light.png#gh-light-mode-only)
```

**Comment by jamessouth** [comment]
[This wiki](https://github.com/jamessouth/polybar-nightlight/wiki) does this when it loads from clicking the `Wiki` link at the top of the page, but when you navigate to `Home` from the sidebar, which is the same URL, the dark mode images load as expected. I used the new syntax:
```markdown
<picture>
	<source media="(prefers-color-scheme: dark)" srcset="https://raw.githubusercontent.com/jamessouth/polybar-nightlight/master/imgs/letter_i_dark.png">
	<img align="left" alt="i" src="https://raw.githubusercontent.com/jamessouth/polybar-nightlight/master/imgs/letter_i.png">
</picture>
```

![good](https://github.com/community/community/assets/25541381/235d0866-2718-4b17-bbf5-78002656ecd0)
![bad](https://github.com/community/community/assets/25541381/92face10-231f-4991-b362-1bef398d73eb)
